fe/src/pages/Error/Error.tsx

```diff
@@ -1,12 +1,13 @@
+import { Box } from '@chakra-ui/react';
 import { Link } from 'react-router-dom';
 
 function Error(): JSX.Element {
   // @TODO - Add some actual content here.
   return (
-    <div>
+    <Box h="100vh">
```

Comment:
I think it's a bad idea to relay on other elements to be 100vh in order for the footer to stay on the bottom. Maybe create a container element in the `/App.tsx` and make that 100vh (also might as well assign a smaller width as well) and then place the router paths in it. This way anything that will be added in the future will stay in the container.

Or just float the footer to the bottom

Reply:
https://jsfiddle.net/wnLT7/6/